### ISSUE TITLE
Finish off the Harptos calendar

### ISSUE THREAD

**Issue by rowanmanning** [rowanmanning/dream-date#1]
What needs doing:

- [ ] Add in [year names from here](http://www.editthis.info/wiki/Year_Names_in_Dalereckoning)
- [x] Find some reference which names a year when a Shieldmeet was held, so that we're certain that the leap years are correct

**Comment by quarterto** [comment]
> There were Shieldmeets every four years.[1] Since 1372 DR contained a Shieldmeet,[5] this meant that 0 DR and every fourth year prior or afterward also contained a Shieldmeet.

http://forgottenrealms.wikia.com/wiki/Shieldmeet#Shieldmeet_Years

**Comment by rowanmanning** [comment]
Ugh wait, they have a year zero!? Please god no

**Comment by quarterto** [comment]
Also negative years 😬

**Comment by quarterto** [comment]
That _should_ Just Work™ with

```js
periods: [
		{
			name: 'Dale Reckoning',
			abbr: 'DR',
			startYear: -700,
		}
]
```

right?

**Comment by rowanmanning** [comment]
Yeah I decided not to represent negative dates for now and tackle that later if anyone is trying to run a historical campaign in Faerûn. I'll verify whether negative start years for periods work later
